### PR TITLE
Update README.md of web-service-sample

### PR DESCRIPTION
**Pull request by insularZJ**

### CODE REVIEW

File: apps/web-service-sample/README.md [hunk 1]
```diff
@@ -1,51 +1,59 @@
...
+
+* [20 Newsgroup dataset(0news-18828.tar.gz)](http://qwone.com/~jason/20Newsgroups/20news-18828.tar.gz) which contains 20 categories and with 19997 texts in total.
+* [GloVe word embeddings(glove.6B.zip)](http://nlp.stanford.edu/data/glove.6B.zip): embeddings of 400k words pre-trained on a 2014 dump of English Wikipedia.
+
+You need to prepare the data by yourself beforehand. The following scripts we prepare will serve to download and extract the data:
+
+    bash ${ANALYTICS_ZOO_HOME}/bin/data/news20/get_news20.sh dir
+    bash ${ANALYTICS_ZOO_HOME}/bin/data/glove/get_glove.sh dir
 
+where `ANALYTICS_ZOO_HOME` is the `dist` directory under the Analytics Zoo project and `dir` is the directory you wish to locate the downloaded data. If `dir` is not specified, the data will be downloaded to the current working directory. 20 Newsgroup dataset and GloVe word embeddings are supposed to be placed under the same directory.
```
Comment (hkvision) [inline apps/web-service-sample/README.md]: modify this according to the lastest master code of TextClassification README

File: apps/web-service-sample/README.md [hunk 2]
```diff
@@ -1,51 +1,59 @@
...
-```
-### Run Application
-Run the Application.java, the web service application will be started. 
-Then, from the browser end, when posted the texts as the request body, the application will respond with the prediction result.
+In the IDE(eg: IDEA), Select New - Project from Existing Source, look through the directory to find pom.xml file of the project(eg: text-classification-training) and click OK, then select open as project in the window pop out next, using maven to build up the project.
+
+### Prepare Data
+The data used in this example are:
+
+* [20 Newsgroup dataset(0news-18828.tar.gz)](http://qwone.com/~jason/20Newsgroups/20news-18828.tar.gz) which contains 20 categories and with 19997 texts in total.
```
Comment (hkvision) [inline apps/web-service-sample/README.md]: 18828 (for the error in the TextClassification readme, I have a PR to update this typo)

**Comment by glorysdj** [comment]
LGTM

**Comment by glorysdj** [comment]
LGTM

**Pull request merged**